web/src/js/ducks/flows.js

```diff
@@ -117,6 +117,17 @@ export function update(flow, data) {
     return { type: REQUEST_ACTION }
 }
 
+export function update_content(flow, file) {
```

Comment:
Let's stick to JS's camel-case and call this `updateContent`. Looks good, but should get an additional "request"/"response" argument :wink: